README.md

```diff
@@ -149,7 +149,7 @@ console.log(chalk.blue('Hello world!'))
 
 ### `fs` package
 
-The [fs](https://nodejs.org/api/fs.html) package available without importing 
+The [fs](https://nodejs.org/api/fs.html) package available is without importing 
```

Comment:
This looks no right. Is it?

Reply:
Good catch.  Making the update in a few.